cl/opinion_page/forms.py

```diff
@@ -250,141 +193,236 @@ def __init__(self, *args, **kwargs) -> None:
-            "second_judge",
-            "third_judge",
-            "panel",
-        ]:
-            self.fields[field_name].queryset = q_judges  # type: ignore[attr-defined]
-            self.fields[field_name].label_from_instance = self.person_label  # type: ignore[attr-defined]
+        :param required: field is required or not
+        :return: None
+        """
+        lead_author = forms.ModelChoiceField(
+            queryset=Person.objects.none(),
+            required=required,
+            label="Lead Author",
+            widget=forms.Select(
+                attrs={
+                    "class": "form-control",
+                    "placeholder": "Choose Lead Author",
+                }
+            ),
+        )
+        self.fields["lead_author"] = lead_author
 
-        if self.pk == "tennworkcompcl":
-            self.fields["cite_reporter"].widget = forms.Select(
-                choices=[("TN WC", "TN WC")],
-                attrs={"class": "form-control"},
-            )
-            self.drop_fields(
-                [
-                    "date_argued",
-                    "date_reargued",
-                    "panel",
-                    "second_judge",
-                    "third_judge",
-                ]
-            )
+    def add_author_str_field(self, required=True):
+        """Add author str field to form
 
-        elif self.pk == "me":
-            self.fields["cite_reporter"].widget = forms.Select(
-                choices=[("ME", "ME")],
-                attrs={"class": "form-control"},
-            )
-            self.drop_fields(["lead_author", "second_judge", "third_judge"])
-        elif self.pk == "tennworkcompapp":
-            self.fields["cite_reporter"].widget = forms.Select(
-                choices=[("TN WC App.", "TN WC App.")],
-                attrs={"class": "form-control"},
-            )
-            self.drop_fields(["date_argued", "date_reargued", "panel"])
-        else:
-            raise BaseException
+        :param required: field is required or not
+        :return: None
+        """
+        author_str = forms.CharField(
+            label="Author String",
+            required=required,
+            widget=forms.TextInput(
+                attrs={
+                    "class": "form-control",
+                    "placeholder": "Use this in case you can't find the name in the "
+                    "list above",
+                    "autocomplete": "off",
+                }
+            ),
+        )
+        self.fields["author_str"] = author_str
+
+    def add_judges_field(self, required=True):
+        """Add judges field to form
+
+        :param required: field is required or not
+        :return: None
+        """
+        judges = forms.CharField(
+            label="Judges",
+            required=required,
+            widget=forms.TextInput(
+                attrs={
+                    "class": "form-control",
+                    "placeholder": "Judges",
```

Comment:
I think a better explanation of what judges is could be helpful.  

something similar to what the model says 

The judges that participated in the opinion as a simple "
            "text string. This field is used when normalized judges "
            "cannot be placed into the panel field.  not this exactly but something clearer.